app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -30,12 +31,19 @@ def connect(options = {})
     # Create the underlying connection according to the version of the oVirt API requested by
     # the caller:
     connect_method = "raw_connect_v#{version}".to_sym
-    connection = self.class.public_send(connect_method, server, port, path, username, password, service)
+    unless @connection.nil?
```

Comment:
right, but it worse to have a connection per call.

we can do a test call. before retrieving the connection.

@jhernand any thoughts about some test call, or ping call via the sdk?
or there is any reference for something like "isClosed" ?